<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>optermdlg</class>
 <widget class="QDialog" name="optermdlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>操作术语配置窗</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../mainwindow.qrc">
    <normaloff>:/image/config.png</normaloff>:/image/config.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="maximumSize">
      <size>
       <width>270</width>
       <height>16777215</height>
      </size>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
     <layout class="QVBoxLayout" name="verticalLayout_5"/>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>术语类型</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,1,2">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>19</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>19</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>宋体</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>操作动作</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="operatorCombo">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>45</width>
              <height>22</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>宋体</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>保护名</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="protectCombo">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>19</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>19</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>宋体</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>状态与位置</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="stateCombo">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>45</width>
              <height>22</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>宋体</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>专用词</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="dictCombo">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>19</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>19</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>宋体</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>特殊符号</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="specialCombo">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>218</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="title">
        <string>术语定义</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="minimumSize">
             <size>
              <width>61</width>
              <height>21</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>61</width>
              <height>21</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>术语组名：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="GroupLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="minimumSize">
             <size>
              <width>61</width>
              <height>21</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>61</width>
              <height>21</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>操作术语：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="operatorLineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="termTable"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="defaultBtn">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>26</height>
          </size>
         </property>
         <property name="text">
          <string>使用默认值</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeBtn">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>26</height>
          </size>
         </property>
         <property name="text">
          <string>关闭</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
